cdap-docs/developers-manual/source/building-blocks/workers.rst

```diff
@@ -0,0 +1,71 @@
+.. meta::
+:author: Cask Data, Inc.
+    :copyright: Copyright © 2014 Cask Data, Inc.
+
+.. _workers:
+
+============================================
+Workers
+============================================
+
+Workers can be used to run background processes. Workers provide an ability to write data processing
```

Comment:
Workers can be used to run background processes. Workers provide the ability to write data processing logic that doesn't fit into the other paradigms such as Flows and MapReduce or realtime and batch.